torch/csrc/jit/passes/create_autodiff_subgraphs.cpp

```diff
@@ -118,6 +132,11 @@ class SubgraphSlicer {
     if (node->kind() == prim::Constant) {
       return false;
     }
+    // view ops as outputs of differentiable subgraphs can cause incorrect differentiation
+    // for now, do not include them in the subgraph
```

Comment:
clang-format want to format it like that
```suggestion
    // view ops as outputs of differentiable subgraphs can cause incorrect
    // differentiation for now, do not include them in the subgraph
```